components/LocationInput.tsx

```diff
@@ -66,27 +63,40 @@ const LocationInput: FC<LocationInputProps> = ({
 
       let popup: mapboxgl.Popup | null = null;
 
+      const renderFunction = (item: MapboxGeocoder.Result) => {
+        var placeName = item.place_name.split(',');
+        return canEdit
+          ? '<div class="mapboxgl-ctrl-geocoder--suggestion"><div class="mapboxgl-ctrl-geocoder--suggestion-title">' +
+              placeName[0] +
+              '</div><div class="mapboxgl-ctrl-geocoder--suggestion-address">' +
+              placeName.splice(1, placeName.length).join(',') +
+              '</div></div>'
+          : '';
```

Comment:
This is the default Mapbox render function

Reply:
Is it exported by them in any way we can use without defining ourselves?

Reply:
if it is, they're doing a great job of hiding it from me. 

the closest thing I could find was the `getRenderFunction()` method, which returned this in a format that wasn't conducive to adding a conditional render function change—I actually had to grab the divs in here from that returned string.

Reply:
Also to clarify—I guess my initial comment was wrong. This is not the mapbox default render function—the ternary returns what the mapbox default render function returns upon true.